Backpack/src/main/java/net/skyscanner/backpack/text/BpkTextField.kt

```diff
@@ -102,21 +103,36 @@ open class BpkTextField @JvmOverloads constructor(
     }
 
     this.iconTintColor = iconColor
-    setTextColor(textColor)
+    setTextColor(
+      colorStateList(
+        color = textColor,
+        pressedColor = textColor,
+        focusedColor = textColor,
+        activatedColor = textColor,
+        disabledColor = textColorDisabled,
+      )
+    )
     setHintTextColor(
-      ColorStateList(
-        arrayOf(intArrayOf(android.R.attr.state_focused), intArrayOf()),
-        intArrayOf(hintFocusedColor, hintNormalColor)
+      colorStateList(
+        color = hintNormalColor,
+        pressedColor = hintNormalColor,
+        focusedColor = hintFocusedColor,
+        activatedColor = hintFocusedColor,
+        disabledColor = textColorDisabled,
       )
     )
 
-    val padding = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
+    val paddingHorizontal = resources.getDimensionPixelSize(R.dimen.bpkSpacingBase)
+    val paddingVertical = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
```

Comment:
We don't have any tokens for 14dp and we cannot get this value using sum of the tokens, therefore we set the closest approximation (8 + 4 = 12dp) for vertical padding. This value will be used only when text inside the box is stretching the box height.